src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -711,24 +690,6 @@ public Color BorderlineColor
             }
         }
 
-        /// <summary>
-        /// The color selector for the borderline of the View.
-        /// Like BackgroundColor, color selector typed BorderlineColor should be used in ViewStyle only.
-        /// So this API is internally used only.
-        /// </summary>
-        internal Selector<Color> BorderlineColorSelector
```

Comment:
As I know, BorderlineColorSelector added due to the APP side request.
@Jaehyun-Cho  Could you check this is removable?

Reply:
I'm afraid it is used in `ViewStyle` class. Please check `ViewStyle.BorderlineColorSelector`.
When `ApplyStyle()`, all properties in ViewStyle class are copied to the corresponding View.
You need to remove both `View.BorderlineColorSelector` and `ViewStyle.BorderlineColorSelector`, or none of them.
But please note that `ViewStyle.BorderlineColorSelector` is currently used in `Tizen.NUI.Components.DefaultThemeCreator.Create()` to decorate outlined button.

Reply:
Hi, Ms. Yang
Because the View doesn't has the property BorderlineColorSelectorProperty, so it will not copy from ViewStyle to View.

Reply:
Hi, Ms. Yang
Because the View doesn't has the property BorderlineColorSelectorProperty, so it will not copy from ViewStyle to View.

Reply:
`ViewStyle.BorderlineColorSelector` is currently used in `Tizen.NUI.Components.DefaultThemeCreator.Create()` to decorate outlined button.
As fas as I know, if you remove `View.BorderlineColorSelector`, the look of outlined button will change because the value of `ViewStyle.BorderlineColorSelector will not copy to the `View.BorderlineColorSelector.